blocks/header/__background-img/_name/header__background-img_name_yellow-tree.css

```diff
@@ -0,0 +1,18 @@
+.header__background-img_name_yellow-tree {
```

Comment:
The background pattern for the header can be imported as one image and included via `background-image`. You don't need to create a separated block for each part of image

Reply:
If I only import one image, how to I edit the image for the different screen widths? On the smallest screen size the trees are much closer to the window than the largest screen. Should I download multiple background images and change the background depending on the screen size?